test/utils/UrlUtilsTest.ts

```diff
@@ -17,6 +17,27 @@ export function UrlUtilsTest() {
       expect(url).toBe('https://a.com/b/c/d');
     });
 
+    it('should support relative urls with leading slash', () => {
+      const url = UrlUtils.normalizeAsString({
+        paths: ['/a', '/b', '//c', '///d']
+      });
+      expect(url).toBe('/a/b/c/d');
+    });
+
+    it('should support relative urls with double leading slash', () => {
+      const url = UrlUtils.normalizeAsString({
+        paths: ['//a', '/b', '//c', '///d']
+      });
+      expect(url).toBe('//a/b/c/d');
+    });
+
+    it('should support relative url with ..', () => {
+      const url = UrlUtils.normalizeAsString({
+        paths: ['../a', '/b', '//c', '///d']
+      });
+      expect(url).toBe('../a/b/c/d');
```

Comment:
Just want to check if the source code for handling leading periods in this PR? Also, is it important to test for relative paths that use a single period `./`?